### ISSUE TITLE
Repackaging

### ISSUE THREAD

**Issue by Digote** [AdmiralCurtiss/YggdraDecode#1]
Do you intend to do the repackaging part of the files? I want to do the localization for my language. Congratulations on the tool; it works perfectly with the data_en.bin.

**Comment by AdmiralCurtiss** [comment]
Wasn't really planning on it, but I suppose it wouldn't be very hard to add, so yeah maybe.

**Comment by AdmiralCurtiss** [comment]
Packing support is now implemented. The code could use a bit of cleaning up, but eh, it should work. Open a new issue if you run into problems.

**Comment by Digote** [comment]
I just tested it, and it seems to work perfectly. I tested by simply repacking it, then changing one byte, and finally making more changes. The first two worked, but the last one didn't. It might be something related to this binary text file; it shouldn't be too complicated to fix.

Once again, thank you very much for your help.

![1](https://github.com/AdmiralCurtiss/YggdraDecode/assets/45742711/4c447b7b-ec4d-4dd4-b520-f0eda87d7511)
